src/Views/reportview.html

```diff
@@ -241,7 +241,7 @@ <h3 class="reportChartHeader">Estimated Flows</h3>
                                                             <tr>
                                                                 <th>Date</th>
                                                                 <th>Estimated</th>
-                                                                <th>{{r.referanceGage.name}}</th>
+                                                                <th>Observed ([{{r.referanceGage.stationID}}])</th>

```

Comment:
Remove the brackets so that it says "Observed (05484500)". Sorry for the lack of clarity in the issue!